.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,187 @@
+name: benchmarks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled # benchmarks label required
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  build:
+    if: github.event_name == 'push' ||
+      github.event_name == 'workflow_dispatch' ||
+      contains(github.event.pull_request.labels.*.name, 'benchmarks')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Restore Atom Data
+        uses: actions/cache/restore@v3
+        id: atom-data-restore
+        with:
+          path: benchmarks/kurucz_cd23_chianti_H_He.h5
+          key: atom-data
+
+      - name: Download Atom Data
+        if: steps.atom-data-restore.outputs.cache-hit != 'true'
+        run: |
+          REPO_URL="https://dev.azure.com/tardis-sn/TARDIS/_apis/git/repositories/tardis-refdata"
+          wget -q "$REPO_URL/items?path=atom_data/kurucz_cd23_chianti_H_He.h5&versionType=branch&version=master&resolveLfs=true" -O benchmarks/kurucz_cd23_chianti_H_He.h5
+
+      - name: Save Atom Data
+        uses: actions/cache/save@v3
+        if: ${{ steps.atom-data-restore.cache-hit != 'true'  && always() || false }}
+        id: atom-data-cache-save
+        with:
+          path: benchmarks/kurucz_cd23_chianti_H_He.h5
+          key: atom-data
+
+      - name: Install TARDIS dependancies
+        run: |
+          LOCK_URL="https://github.com/tardis-sn/tardis/releases/download/release-2023.06.18/conda-linux-64.lock"
+          wget -q "$LOCK_URL"
+
+      - name: Install asv
```

Comment:
isn't that part of the tardis env

Reply:
What is?

Reply:
`asv`

Reply:
Nope, it's not in there.

Reply:
Yes, it builds the env each time so no need to add it there.